inscriptions/ISic0852.xml

```diff
@@ -155,44 +159,34 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" subtype="PHI" n="PHI140329">
+            <div type="edition" xml:lang="grc">
                 <ab>
-                    <lb n="1"/>
-
-
-                     
-                    <expan>
-                        <abbr>Θ</abbr>
-                        <ex>εοῖς</ex>
-                    </expan>
-                    <expan>
-                        <abbr>Κ</abbr>
-                        <ex>αταχθονίοις</ex>
-                    </expan>
-                    .<lb n="2"/>ΠΙΡΟΣΙΦΙΤΟΥ {²⁷<supplied reason="omitted">Π</supplied>οσ
-                    <unclear>τ</unclear>
-                    <supplied reason="omitted">ούμι</supplied>ο
-                    <supplied reason="omitted">ς</supplied>
-                    <certainty match="preceding-sibling::node()" locus="value"/>}²⁷<lb n="3"/>  Δεξιὸς<lb n="4"/>ΚΑΙΣΩΟΔ<supplied
-                        reason="lost">— —
-                </supplied> {²⁷Κ<supplied reason="omitted">λ</supplied>(αυδίᾳ) Σω
-                    <supplied reason="omitted">φρ</supplied>
-                    <supplied reason="lost">οσύνῃ</supplied>
-                    <certainty match="preceding-sibling::node()" locus="value"/>}²⁷
-                    <lb n="5"/>
-                    <expan>
-                        <abbr>ἔζησ</abbr>
-                        <ex>εν</ex>
-                    </expan>
-                    <supplied reason="lost">ἔτη — — — — —</supplied>
-                    <lb n="6"/>
+                    <lb n="1"/><expan><abbr>Θ</abbr><ex>εοῖς</ex></expan> <expan><abbr>Κ</abbr><ex>αταχθονίοις</ex></expan>
+                    <lb n="2"/><orig>πιροσιφιτου</orig>
+                    <lb n="3"/>δεξιὸς
+                    <lb n="4"/><orig>καισωο</orig>
+                    <lb n="5"/><orig>εζησης</orig>
                 </ab>
             </div>
             <div type="apparatus">
-                <p><!--external apparatus criticus (if applicable)--></p>
+                <listApp>
+                    <app loc="line2">
+                        <note>Π <supplied reason="omitted">Π</supplied>οσ<unclear>τ</unclear><supplied reason="omitted">ουμι</supplied>ος, Keibel</note>
+                    </app>
+                    <app loc="line4">
+                        <note><expan><abbr>Κ<supplied reason="omitted">λ</supplied><ex>αύδια</ex></abbr></expan> Σω<supplied reason="omitted">φρ</supplied><supplied reason="lost" cert="low">οσύνῃ</supplied>, Keibel</note>
+                    </app>
+                    <app loc="line5">
+                        <note><expan><abbr>ἔζης</abbr><ex>εν</ex></expan><supplied reason="lost">ἔτη</supplied></note>
+                    </app>
+                </listApp>
+                <p>Keibel's restorations are just tentative, cf. Korhonen 2007 p. 294</p>
```

Comment:
kaibel; we need statement of the source text here - kaibel? For the Apparatus, full mark-up not strictly necessary.